wls-exporter-core/src/main/java/com/oracle/wls/exporter/domain/ExporterConfig.java

```diff
@@ -3,27 +3,21 @@
 
 package com.oracle.wls.exporter.domain;
 
-import java.io.InputStream;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.stream.Stream;
-
 import com.google.gson.JsonObject;
 import org.yaml.snakeyaml.Yaml;
 import org.yaml.snakeyaml.constructor.SafeConstructor;
 import org.yaml.snakeyaml.scanner.ScannerException;
 
+import java.io.InputStream;
+import java.util.*;
```

Comment:
Minor point... Did something change in the Sonar settings to allow starred imports?

Reply:
It is possible that the WME Sonar settings may be different for WME?

Reply:
I think I have fixed it